client/my-sites/plans/jetpack-plans/slug-to-selector-product.ts

```diff
@@ -166,7 +165,7 @@ function itemToSelectorProduct(
 		}
 
 		// We do not support TERM_BIENNIALLY or TERM_TRIENIALLY for Jetpack plans
```

Comment:
Not a blocker, but we should update this comment and remove `TERM_BIENNIALLY` from it.

Reply:
Good catch, thank you